article/sections/_7_latent_confounding.tex

```diff
@@ -1,5 +1,224 @@
+interest exogenous. The process of applying the method is actually quite 
+straightforward and simple. It proceeds as follows:
+\begin{itemize}
+	\item First, estimate the substitute confounder using any good latent variable 
+	model the modeler chooses. The authors suggest estimating a factor model 
+	with k factors on the set of covariates the modeler is interested in. 
+	\item Second, check the factor model's accuracy using posterior predictive 
+	checks (add details). 
+	\item Once a sufficiently accurate latent variable model is recovered, use it 
+	to estimate an expected value of the latent variable for each observation, 
+	and control for this value in the outcome model, alongside the treatment 
+	variables and other covariates of interest. 
+\end{itemize}
+
+
+
+One main assumption of the deconfounder algorithm is that the data at hand 
+should have multi-cause confounders, thus the title of the paper, "The 
+blessings of multiple causes". In other words, this method works when the 
+unobserved confounders affect multiple of the observed causes (or treatment 
+variables) of interest, alongside the outcome. This assumption is weaker than 
+the one required for ignorability to hold, which requires the absence of both 
+single cause and multi-cause confounders for accurate causal inferences. 
+
+
+
+\subsection{Case study: Simulation}
+\label{sec:deconfounder-simulation}
+
+The purpose of this section is to investigate the effectiveness of the 
+deconfounder algorithm \citep{wang_2019_blessings} in adjusting for unobserved 
+confounding. We use a simulated mode choice data where travel distance 
+linearly confounds both travel time and travel cost. We then mask the travel 
+distance data and treat it as an unobserved variable. 
+
+
+We estimate three models:
+
+\begin{itemize}
+	\item Model 1: A multinomial logit with the correct original specification, 
+	EXCEPT we ommit the travel distance variable in the specification without 
+	trying to adjust for it. This model represents the worst case scenario 
+	where a modeler ignores, or is unaware of, unobserved confounding.
+	\item Model 2: We use the deconfounder algorithm to try to recover the 
+	confounder (travel distance). In this method, we use all the variables in 
+	each mode's utility to recover that mode's confounder. This is in line 
+	with the approach taken in \citet{wang_2019_blessings}, where they use all the set 
+	of observed variables in the factor model to recover a substitute 
+	confounder.
+	\item Model 3: We use the deconfounder algorithm to try to recover the 
+	confounder (travel distance), but this time, we only use travel time and 
+	cost in the factor model, instead of all the variables in the utility 
+	specification of each mode. By only using what we know are confounded 
+	variables to recover the substitute confounder, our goal is to analyze 
+	whether we can improve the accuracy of this approach by adopting a more 
```

Comment:
Since including extra variables won't bias your factor analysis inferences (e.g. asymptotically, you'll just recover factor loading of zero from irrelevant variables), it is just reducing variance (less params to estimate). Lower variance estimator = higher efficiency estimator (less data needed to get good estimates).
```suggestion
	whether we can improve the efficiency of this approach by adopting a more 
```